src/test/java/project/web/GasStationControllerTest.java

```diff
@@ -33,11 +33,15 @@ void getGasStationByNameTestWithValidInput() {
         List<GasStationDTO> stations = setupResultsFromGetStationByNameToReturnListWithOneStation(stationName);
 
         // WHEN
-        ResponseEntity<List<GasStationDTO>> response = gasStationController.getGasStationByName(stationName);
+        ResponseEntity<List<GasStationDTO>> gasStationByName = gasStationController.getGasStationByName(stationName);
```

Comment:
jfi: i think, the naming here can be better, imho "ByName" is just too much. Why i need to know, how you got this object?  Also variables with `DTO` as suffix are imho not really great.

Reply:
accepted and fixed :)